src/ui/public/chrome/directives/app_switcher/app_switcher.js

```diff
@@ -6,14 +6,45 @@ var DomLocationProvider = require('ui/domLocation');
 
 require('ui/modules')
 .get('kibana')
+.provider('appSwitcherEnsureNavigation', function () {
+  var forceNavigation = false;
```

Comment:
Consider this comment for all `var`s in this file rather than me duplicating it a bunch.